### PR TITLE
update README

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add the updated README in the development branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,13 +16,14 @@ Inclusive Mart is an e-commerce platform for marvel fans, Get Your Best Deal.
...
-- Products (https://inclusive-mart.netlify.app/pages/products.html)
-- Cart Management (https://inclusive-mart.netlify.app/pages/cart.html)
-- Wishlist Management (https://inclusive-mart.netlify.app/pages/wishlist.html)
-- Log In (https://inclusive-mart.netlify.app/pages/signin.html)
-- Sign Up (https://inclusive-mart.netlify.app/pages/signup.html)
+Currently, it contains screens for the following pages
+
+- Home (<a href="https://inclusive-mart.netlify.app/" target="_blank">Inclusive Mart Home</a>)
+- Products (<a href="https://inclusive-mart.netlify.app/pages/products.html" target="_blank">Inclusive Mart Products</a>)
+- Cart Management (<a href="https://inclusive-mart.netlify.app/pages/cart.html" target="_blank">Inclusive Mart Cart</a>)
```
Comment (anik31) [inline README.md]: target="_blank" is not supported in github markdown, so you can remove it and instead use the markdown syntax for links.

Like this -
image

**Review by anik31 — COMMENTED** [review]
Nice work with the readme, it looks good 💯 

Suggestion - put the demo at the top instead of bottom

**Review by sainath7878 — APPROVED** [review]
Make the changes as mentioned by Aniket. Rest looks good. Great work!

File: README.md (hunk 1 above)
Comment (hsnice16) [reply README.md]: Oh! okay. Thanks for letting me know.

**Comment by hsnice16** [comment]
Thank you for reviewing this.

**Pull request merged**